### DISCUSSION TITLE
CefSharp.BrowserSubprocess crashes

### DISCUSSION THREAD

**Discussion by assopri** [cefsharp/CefSharp#3584] [Q&A]
I get the following messages during my multithreaded app is working (each thread using Winforms Cefsharp browser with separate cahce). Any ideas on reasons or how can I go deeper to research this issues better?

![image](https://user-images.githubusercontent.com/73235464/120168485-94169680-c207-11eb-87a7-a92cec3c2e7a.png)

My OS is Windows7 64x , VS 2019 installed. I use Winforms Cefsharp version, .NET 4.5.2 . Cef version is 90.6.50.0

**Comment by assopri** [comment]
One more thing - any workaround to somehow just hide this message?

**Reply by amaitland** [reply]
As it's libcef.dll that's the origin of the crash you'll need to download libcef.dll.pdb, instructions at https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#loading-native-symbols-for-easier-diagnosis

If you enable crash reporting the dialog should be suppressed and a mini dump created instead. 

https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#crash-reporting

**Reply by assopri** [reply]
I found the problem reason - it was parallel loading of localhost http://localhost
